src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -124,10 +124,11 @@ public OrganizationUsersController(
     }
 
     [HttpGet("{id}")]
+    [Authorize<ManageUsersRequirement>]
     public async Task<OrganizationUserDetailsResponseModel> Get(Guid id, bool includeGroups = false)
     {
         var (organizationUser, collections) = await _organizationUserRepository.GetDetailsByIdWithCollectionsAsync(id);
-        if (organizationUser == null || !await _currentContext.ManageUsers(organizationUser.OrganizationId))
+        if (organizationUser == null)
         {
             throw new NotFoundException();
         }
```

Comment:
You still need to make sure the orgUser belongs to the organization specified in the route, otherwise you could mix & match an organization ID that you own with an unrelated orgUser.